bfrescox_pypkg/src/bfrescox/__init__.py

```diff
@@ -7,23 +7,25 @@
 
 from importlib.metadata import version
 
+from . import parse_differential_xs
 from ._run_frescox_simulation import (
+    FRESCOX_COREX_SUPPORT,
     FRESCOX_EXE,
-    FRESCOX_MPI_SUPPORT, FRESCOX_OPENMP_SUPPORT,
     FRESCOX_LAPACK_SUPPORT,
-    FRESCOX_COREX_SUPPORT
+    FRESCOX_MPI_SUPPORT,
+    FRESCOX_OPENMP_SUPPORT,
 )
-
+from .Configuration import Configuration
+from .generate_elastic_template import generate_elastic_template
+from .generate_inelastic_template import generate_inelastic_template
 from .information import information
+from .load_tests import load_tests
+from .parse_fort16 import parse_fort16
+from .parse_parallelization_setup import parse_parallelization_setup
+from .parse_performance_results import parse_performance_results
 from .print_information import print_information
 from .run_simulation import run_simulation
 
-from .Configuration import Configuration
-
-# ----- Python unittest-based test framework
-# Used for automatic test discovery
```

Comment:
These comments and this grouping seem important.  `load_tests` is somehow different from all the other functions in the interface.  In fact, I wish I could move it to the private interface, but don't think that I can.

Ditto for `bfrescoxpro`.